vertical-pod-autoscaler/pkg/recommender/input/spec/spec_client.go

```diff
@@ -76,6 +76,12 @@ func (client *specClient) GetPodSpecs() ([]*BasicPodSpec, error) {
 	}
 	return podSpecs, nil
 }
+
+// IsActive returns true if Pod is in active phase.
+func (pod *BasicPodSpec) IsActive() bool {
+	return pod.Phase != v1.PodSucceeded && pod.Phase != v1.PodFailed
```

Comment:
What exactly do you want to be covered by "active" phase? Is it a state where we still expect usage samples from the Pod's contaners?
Looking at https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-phase I'd expect at least Completed to be excluded here too.

Reply:
I'd expect it too if it existed :) It seems the docs are not in sync with the api: https://github.com/kubernetes/api/blob/7b2abe14437878fa82cec898c0cd89bd09ad5250/core/v1/types.go#L2380

Reply:
Removed anyway as I moved away from checking if matched pods are active per offline chat.